src/app/helpers/telemetryHelper.js

```diff
@@ -217,15 +217,18 @@ module.exports = {
   /**
    * This function helps to log API access event
    */
-  logAPIAccessEvent: function (req) {
-    const apiConfig = telemtryEventConfig.URL[req.uri] || {}
-
+  logAPIAccessEvent: function (req, result, uri) {
+    req = telemetry.getTelemetryAPISuceessData(req, result, uri)
+    req.channel = req.channel ? req.channel : envHelper.DEFAULT_CHANNEL
+    const apiConfig = telemtryEventConfig.URL[uri] || {}
     let object = {}
     let params = []
-    if (req.options) {
-      params = this.getParamsData(req.options, req.statusCode, req.resp, req.uri)
+    if (req.reqObj) {
+      params = this.getParamsData(req.reqObj, req.statusCode, req.resp, uri)
     }
-    const edata = telemetry.logEventData('api_access', 'INFO', apiConfig.message, params)
+
+    const edata = telemetry.logEventData('api_access', 'INFO', apiConfig.message,JSON.stringify(params))
```

Comment:
use constants instead of using hard coded strings

Reply:
ok